booster_api/features/src/support/planner_jsons/add_wi_comment.json

```diff
@@ -0,0 +1 @@
+{"data":{"type":"comments","attributes":{"body":"$comment_text","markup":"Markdown"},"links":{}}}
```

Comment:
You probably don't need the directory _planner_jsons_

Reply:
Create a new directory _booster_jsons_ and use that instead.

Reply:
Good idea - done!